.env.example

```diff
@@ -4,6 +4,7 @@ VUE_APP_PATH="/"
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
 VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
+VUE_APP_CORPORATE_ONLINE_URL="https://www.corporateonline.gov.bc.ca"
```

Comment:
The VUE_APP_CORPORATE_ONLINE_URL variable is the only one without a trailing slash. Is this intentional?

Reply:
Let me double-check that. I copied it from Auth Web.

Reply:
I think it's fine since we're only going to open that link in a new window. (It's not an API and we're not going to add a route.)

Good catch and question 👍

Reply:
It's the same as here so I'm assuming it's OK? https://github.com/bcgov/sbc-auth/blob/c832d083ca738d8c297ff8f5364a214d0ba5dae5/auth-web/.env.example#L27